### ISSUE TITLE
Can't find the downloaded files

### ISSUE THREAD

**Issue by realmzakaria** [mimbrero/whatsapp-desktop-linux#49]
I am using the flatpak version and whenever I download any file the downloaded pop-up appears but can't find the so-called "downloaded" file. Where the downloaded files are stored?

**Comment by JGMelon22** [comment]
Well, should be at `~/Downloads`, at least in theory

**Comment by realmzakaria** [comment]
> Well, should be at `~/Downloads`, at least in theory

No. Files doesn't get stored in that path. That's why I am looking for files.

**Comment by JGMelon22** [comment]
Now I'm facing the same problem as yours :/

**Comment by suaide** [comment]
is this fixed? I can not find downloaded files

**Comment by JGMelon22** [comment]
"Magically" it was fixed after sometime. Might be a worth shot to delete the app cache deleting it at the directory `./var/app/io.github.mimbrero.WhatsAppDesktop` and then run a `sudo flatpak repair` just in case
